test/presenters/publication_presenter_test.rb

```diff
@@ -74,9 +74,11 @@ def schema_name
     assert_equal(presented.national_applicability[:wales][:alternative_url], "http://wales.gov.uk/topics/statistics/headlines/housing2012/121025/?lang=en")
   end
 
-  test "presents the single page notification button" do
-    presented = presented_item("statistics_publication")
-    assert presented.has_single_page_notifications?
+  test "displays the single page notification button on English pages" do
```

Comment:
It would be good to also add at least one test somewhere which checks that, if the locale isn't english, the button isn't displayed.
This applies to both the sign-up button and to the single page notification button (so at least 2 tests).